src/makefiles/linux_atlas.mk

```diff
@@ -24,11 +24,12 @@ CXXFLAGS = -msse -msse2 -Wall -I.. \
       -g # -O0 -DKALDI_PARANOID 
 
 ifeq ($(KALDI_FLAVOR), dynamic)
-CXXFLAGS += -fPIC
+KALDI_CXXFLAGS += -fPIC
 endif
 
-LDFLAGS = -rdynamic $(OPENFSTLDFLAGS)
-LDLIBS = $(EXTRA_LDLIBS) $(OPENFSTLIBS) $(ATLASLIBS) -lm -lpthread -ldl
+CXXFLAGS := $(KALDI_CXXFLAGS) $(CXXFLAGS)
```

Comment:
if we're going to do this I think it's ugly to  have EXTRA_CXXFLAGS be  included in KALDI_CXXFLAGS-- it would be clearer, IMO, to do
CXXFLAGS := $(KALDI_CXXFLAGS) $(EXTRA_CXXFLAGS) $(CXXFLAGS)
and remove EXTRA_CXXFLAGS from KALDI_CXXFLAGS.